File: algorithms/python/firstAndLastPosition.py
class Solution:
    def searchRange(self, nums: List[int], target: int) -> List[int]:
        positions = [-1, -1]
        for i, num in enumerate(nums):
            if positions[0] == -1 and nums[i] == target:
                positions[0] = i
            if target == nums[i]:
                positions[1] = i
        return positions

File: algorithms/java/firstAndLastPosition.java
class Solution {
    public int[] searchRange(int[] nums, int target) {
       int position[] = {-1,-1};


       for(int i = 0; i < nums.length; i++){
           if(position[0] == -1 && target == nums[i]){
               position[0] = i;
           }
           if(target == nums[i]){
               position[1] = i;
           }
       }

       return position;
    }
}